extension/js/common/core/mime.ts

```diff
@@ -208,7 +210,11 @@ export class Mime {
       } else {
         contentNode = new MimeBuilder('multipart/alternative'); // tslint:disable-line:no-unsafe-any
         for (const type of Object.keys(body)) {
-          contentNode.appendChild(Mime.newContentNode(MimeBuilder, type, body[type]!.toString())); // already present, that's why part of for loop
+          let content = body[type]!.toString();
+          if (type === 'text/html') {
+            content = Mime.extractInlineImagesToAttachments(content, rootNode);
```

Comment:
I suspect this mechanism will also apply to pgp/mime messages, which is a mime message containing HTML, then encrypted as a whole and sent out as an attachment.

Since inside that encrypted envelope is a html mime message, I think it would use this mechanism as written.

Which may be a good thing. Just need to check that for example Thunderbird can render it (probably yes).

Reply:
Need to check that FlowCrypt itself can render it (pgp/mime rich text message with inline attachments like this).

Reply:
> I suspect this mechanism will also apply to pgp/mime messages

Not anymore, now it'll be applied only for plain rich-text messages.